nisystemlink/clients/artifact/models/_upload_artifact_request.py

```diff
@@ -0,0 +1,11 @@
+from typing import Optional
+
+from nisystemlink.clients.core import ApiError
+from nisystemlink.clients.core._uplink._json_model import JsonModel
+
+
+class UploadArtifactResponse(JsonModel):

```

Comment:
You have it as response here but the file name is saying as request..

Reply:
Updated the file name.